website/layouts/docs.erb

```diff
@@ -195,7 +195,7 @@
           </li>
 
           <li<%= sidebar_current("docs-commands-providers") %>>
-            <a href="/docs/commands/providers.html">providers</a>
+            <a href="/docs/commands/providers/index.html">providers</a>
```

Comment:
Over in the `terraform-website` repository there's a file `content/redirects.txt` that we can use to redirect `/docs/commands/providers.html` over here.

On the other hand, maybe it'd be simpler to just leave that page where it used to be and use the subdirectory only for the subcommand pages? Not _too_ concerned which way we do it, but at least if we leave the URL unchanged then we don't need to coordinate changes between both repositories to get this done. (If we go the redirect route then we'll need to time the merge in that repository carefully so that the redirect doesn't get deployed before these docs. :confused:  )

Reply:
I get you point, but if we do that it won't have the sidebar with the subcommand. 

I'm all for it if that's ok with you.

Reply:
The layout setting in the markdown file is independent of the filesystem layout, so it should be possible to have it set to your new layout even though it's in the parent directory... just need to make sure all the URLs in the layout file are root-relative so it'll work from both locations.

Reply:
reverted!